src/services/travelServices.js

```diff
@@ -1,18 +1,20 @@
 import 'core-js/stable';
 import 'regenerator-runtime/runtime';
 import models from '../database/models';
+import { async } from 'regenerator-runtime/runtime';
 
-const { travels } = models;
+const { travels, travel_requests, departments, users } = models;
```

Comment:
I don't think is ideal we having two travel request tables. I will suggest you modify the already existing table to fit into the task you are working on, and also change the name from travel to travel_request as agreed by the team.

Reply:
@chuksjoe  that's fine, if we all agree it should be removed.

Reply:
I thought we already agreed to have one request table?